GitUI/BranchTreePanel/SubmoduleNode.cs

```diff
@@ -86,7 +87,21 @@ public void LaunchGitExtensions()
                 return;
             }
 
-            GitUICommands.LaunchBrowse(workingDir: Info.Path.EnsureTrailingPathSeparator(), ObjectId.WorkTreeId, Info?.Detailed?.RawStatus?.OldCommit);
+            ObjectId? first;
+            ObjectId? second;
+            if (!IsCurrent)
+            {
+                first = ObjectId.WorkTreeId;
+                second = Info?.Detailed?.RawStatus?.OldCommit;
```

Comment:
What's the benefit of this old behavior? Why should we have both? Is it clear to the user which one is chosen?
Either remove or add a comment, please.

Reply:
Old behavior?
This adds the selection when current module, the other is when a submodule is clicked, then select the commited commit and the worktree, i.e. the "diff" for the submodule.

Reply:
Thank you for the clarification. I think future readers would appreciate a comment like `open a "diff" of submodule changes`, too.

Reply:
added comment